.github/workflows/build.yml

```diff
@@ -77,3 +77,31 @@ jobs:
           go-version: '1.17.5'
       - run: make generate && ./lint.sh
       - run: cd node && go test -v ./...
+
+  # Run rust lints and tests
+  rust-lint-and-tests:
+    runs-on: tilt-kube-public
```

Comment:
I don't think this needs to run on the tilt kube box. can we just run it on ubuntu-20.04 like the others?